Intersect.Client/Networking/PacketHandler.cs

```diff
@@ -1372,7 +1371,6 @@ public void HandlePacket(IPacketSender packetSender, ProjectileDeadPacket packet
                 {
                     if (Globals.Entities.ContainsKey(projDeath) && Globals.Entities[projDeath] is Projectile projectile)
```

Comment:
you can change `is Projectile projectile` to `is Projectile` -- the variable is no longer used